UN-OCHA/rwint9-site
# RW-129 User pages

- Add user paths to theme switcher rule
- Add form fields to user template override file
- Add admin menu icon
- Bookmarks item styles
- RW-123 Admin menu shows for specific roles and user 1
- Fix taxonomy TOC sticky nav
- Adjust Source term field names


## To test
Import config so the User pages use the `common_design_subtheme`

The admin menu should appear to user 1, and any users with editor, administrator, webmaster roles
